.devops/deploy-pipelines.yml

```
@@ -60,6 +60,16 @@ resources:
       name: pagopa/azure-pipeline-templates
       ref: refs/tags/v6.14.0
       endpoint: 'io-azure-devops-github-ro'
+    - repository: pagopaCheckoutTests
+      type: github
+      name: pagopa/pagopa-checkout-tests
+      ref: CHK-4599-ecommerce-cdc-api-tests
+      endpoint: 'io-azure-devops-github-ro'
+    - repository: pagopaWalletTests
+      type: github
+      name: pagopa/pagopa-wallet-tests
+      ref: CHK-4599-integration-test
```

```suggestion
      ref: main
```

TODO: to be changed after dependent PR merge